includes/tools.php

```diff
@@ -223,3 +226,145 @@ function verify_https_support() {
 	 */
 	return 200 === wp_remote_retrieve_response_code( $response );
 }
+
+/**
+ * Build a system environment report for submission along with any bug reports.
+ *
+ * This is largely based on (a simplified version) of Easy Digital Downloads' edd_tools_sysinfo_get()
+ * reporting function.
+ *
+ * @link https://github.com/easydigitaldownloads/easy-digital-downloads
```

Comment:
That's a really good question — I rewrote a fair portion of it (>50%, so it's more an "inspired by"), and it tears out all of the EDD-specific logic, but the pattern and formatting is still pretty damn similar. I'd like to avoid changing to GPL, if at all possible though, as the Airstory + Liquid Web teams were already in agreement on MIT (and MIT can be used more freely without the viral nature of the GPL).

At what point do you think we cross from being unique work to infringing on Pippin and team, thus invoking the GPL?